### PR TITLE
UAR-1548 Add Status Column LE Trustee Review

### PR DESCRIPTION
**Pull request by mattch1**
### JIRA link

https://companieshouse.atlassian.net/browse/UAR-1548

### Change description

Missing status column added and changed query value in template to reflect the  "still involved" value supplied.

### Work checklist

- [x] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: test/controllers/update/update.manage.trusts.review.legal.entities.controller.spec.ts [hunk 1]
```diff
@@ -203,6 +203,38 @@ describe('Update - Manage Trusts - Review legal entities', () => {
       expect(resp.text).toContain('Beneficiary');
     });
 
+    test('Page displays status as active when entity is still involved in the trust', async () => {
```
Comment (mwestacott) [inline test/controllers/update/update.manage.trusts.review.legal.entities.controller.spec.ts:206]: Please change name from 'entity' to 'legal entity trustee' or 'corporate trustee'. Below also.

File: views/update/update-manage-trusts-review-legal-entities.html [hunk 2]
```diff
@@ -14,14 +14,19 @@
 
 {% set rows=[] %}
 {% for trustee in pageData.legalEntities %}
+  {% set trusteeStatus = 'Removed' if (trustee.still_involved === "No") else 'Active' %}
+  {% set trusteeRole = trusteeMacros.readableTrusteeRole(trustee.type | upper) %}
   {% set changeLink = OE_CONFIGS.UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_LEGAL_ENTITY_URL + "/" + trustee.id %}
   {% set rows = (rows.push(
     [
-      {
+      {
```
Comment (mwestacott) [inline views/update/update-manage-trusts-review-legal-entities.html:22]: Trailing space after { should be deleted.

Comment (mattch1) [reply views/update/update-manage-trusts-review-legal-entities.html]: Removed spaces

File: test/controllers/update/update.manage.trusts.review.legal.entities.controller.spec.ts (hunk 1 above)
Comment (mattch1) [reply test/controllers/update/update.manage.trusts.review.legal.entities.controller.spec.ts]: Renamed

**Review by lduranteau — APPROVED** [review]

**Review by dlloyd1-ch — APPROVED** [review]

**Review by HatiDev — APPROVED** [review]

**Review by mwestacott — APPROVED** [review]

**Pull request merged**